packages/cashier/src/pages/p2p-cashier/__tests__/p2p-cashier.spec.js

```diff
@@ -22,32 +17,148 @@ describe('<P2PCashier />', () => {
     const history = createBrowserHistory();
 
     it('should render <Loading /> component', () => {
+        const mockRootStore = {
```

Comment:
@farzin-deriv could we refactor mockRootStore to avoid duplication? For example create mockRootStore in beforeEach function and in test cases just change the required field. (because e.g in 'should render P2P component' test case we only change `is_logging_in` field and in 'should redirect to "/cashier/p2p" page with "?order=1" query parameter' test case as i can see there is no any difference with original mockRootStore variable 🤔)?

Reply:
@george-usynin-binary Yes you are right, We should have a mocked version of the `RootStore` so we won’t need to create it each time for each test case, I've already created a card for it and added my proposed API for it, We can discuss it and come up with better ideas 🙇🏻